docs/manual/fault_domains.md

```diff
@@ -265,7 +265,9 @@ func (cluster *FoundationDBCluster) DesiredCoordinatorCount() int {
 ```
 
 For all clusters that use more than one region the operator will recruit 9 coordinators.
-If the number of regions is `1` the number of recruited coordinators depends on the redundancy mode:
+If the number of regions is `1` the number of recruited coordinators depends on the redundancy mode.
+The number of coordinators is chosen based on the fact that the coordinators use a consensus protocol (Paxos) that needs a majority of processes to be up.
```

Comment:
Where cluster size = failure tolerance * 2 + 1. In case of triple redundancy we expect cluster to be available losing 2 nodes, so for coordinator we have n  =  2 * 2 + 1

Reply:
can we add this line to documentation?